action.yml

```diff
@@ -7,6 +7,8 @@ inputs:
     default: 'false'
   node-version:
     description: 'Version Spec of the version to use.  Examples: 12.x, 10.15.1, >=10.15.0'
+  node-version-file:
+    description: 'File containing the version Spec of the version to use.  Examples: .nvmrc'
```

Comment:
Will this work natively with other types of files such as `.node-version`? If so the example could be expanded upon a bit

Some more information that I could dig up here: https://github.com/shadowspawn/node-version-usage and https://stackoverflow.com/questions/27425852/what-uses-respects-the-node-version-file

Reply:
It works with `.node-version`  and other similar files.  Updated the example to reflect that. 👍🏾